.github/labeler.yml

```diff
@@ -603,3 +596,22 @@
 # Add Documentation Label
 'Documentation':
   - '*.md'
+
+'CI':
+  - .travis.yml
```

Comment:
We're using travis?

Reply:
I just moved this line from top of the file,

Reply:
doesn't matter then, you can keep it or remove it.